test_nx/nx_g_large_property.e

```diff
@@ -0,0 +1,18 @@
+src,dst,p1,p2,p3,p4,p5,p6,p7,p8,p9,p10,p11,p12,p13,p14,p15
```

Comment:
format is incorrect....

Reply:
a property graph has many files, each file record ONE type of vertices/edges with the same label.
```
# a file named person.v for vertices with label person
vid username gender age ...
1 Jim male 24
2 XX male 20
```

```
# a file named company.v for vertices with label company
vid name location founded_time
1 Tencent SZ 1999
2 Ali HZ 1999
```

```
# a file named works_in.py for edges with label works_in
srcid dstid hire_time
1 1 1999
2 1 2020
```
... so a graph with X-label edges and Y-label vertices should have X+Y files.

Reply:
nx_g_large_property.e and nx_g_large_property.v are used for nx graph test :
```
        gs_g = load_from(vertices={
            'v': ([("vid", VId.INT), ('p1', 'string'), ('p2', 'int'),
                   ('p3', 'string'), ('p4', 'double'), ('p5', 'double'),
                   ('p6', 'int'), ('p7', 'string'), ('p8', 'string'),
                   ('p9', 'string'), ('p10', 'double'), ('p11', 'string'),
                   ('p12', 'float'), ('p13', 'string'), ('p14', 'int'),
                   ('p15', 'double')], loader(vfile_large_property))
        },
                       edges={
                           'e': ('v', 'v', 'both_in_out', [("src", SrcId.INT),
                                                           ("dst", DstId.INT),
                                                           ('p1', 'string'),
                                                           ('p2', 'int'),
                                                           ('p3', 'string'),
                                                           ('p4', 'double'),
                                                           ('p5', 'double'),
                                                           ('p6', 'int'),
                                                           ('p7', 'string'),
                                                           ('p8', 'string'),
                                                           ('p9', 'string'),
                                                           ('p10', 'double'),
                                                           ('p11', 'string'),
                                                           ('p12', 'float'),
                                                           ('p13', 'string'),
                                                           ('p14', 'int'),
                                                           ('p15', 'double')],
                                 loader(efile_large_property))
                       })
```
p1-15 in edge file are different from those in vertex file.
I will make this part more clear.
Should I simplify this part and make these properties more clear?